lms/templates/learner_dashboard/program_header_view.underscore

```diff
@@ -5,7 +5,7 @@
 </picture>
 <h2 class="hd-2 title"><%- name %></h2>
 <p class="subtitle"><%- subtitle %></p>
-<a href="" class="breadcrumb"><%- gettext('Programs') %></a>
+<a href="/dashboard/programs" class="breadcrumb"><%- gettext('Programs') %></a>
```

Comment:
@rlucioni Are you going to add the Program LIsting page url to the json in this PR? Or that would come in another PR? Or you think the hard-coded value `/dashboard/programs` is good enough?